Library/Formula/x265.rb

```diff
@@ -13,20 +14,17 @@ class X265 < Formula
     sha256 "440805e02452b80400ff428967bd4c5d24f042bea2cc6b5d76a3a37648965a60" => :mountain_lion
   end
 
-  head 'https://bitbucket.org/multicoreware/x265', :using => :hg
+  option "with-16-bit", "Build a 16-bit x265 (default: 8-bit)"
 
-  depends_on 'yasm' => :build
-  depends_on 'cmake' => :build
-  depends_on :macos => :lion
+  deprecated_option "16-bit" => "with-16-bit"
 
-  option '16-bit', 'Build a 16-bit x265 (default: 8-bit)'
+  depends_on "yasm" => :build
+  depends_on "cmake" => :build
+  depends_on :macos => :lion
 
   def install
-
     args = std_cmake_args
-    args.delete '-DCMAKE_BUILD_TYPE=None'
-    args << '-DCMAKE_BUILD_TYPE=Release'
-    args << '-DHIGH_BIT_DEPTH=ON' if build.include? '16-bit'
+    args << "-DHIGH_BIT_DEPTH=ON" if build.include? "16-bit"
```

Comment:
There's language in there already about deprecating options, but perhaps we're only applying to the new formulae and changing them as we can for now - That would be in line with our policy on SHA256, double quotes, etc.